### DISCUSSION TITLE
2021-22: 一路向西

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#217] [Announcements]
https://xuanwo.io/reports/2021-22/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by zjuyk** [comment]
羡慕，以前也想过去大理玩，可是没找到合适的机会，看着那野生菌火锅的图就觉得好吃

**Comment by userid** [comment]
要是没有最后半句话，就看起来更赏心悦目，狗粮味骤减~
